templates/automotive/index.html

```diff
@@ -56,16 +56,24 @@ <h3 class="p-heading--2">The best operating system for all your needs</h3>
           <a href="https://www.elektrobit.com/products/ecu/eb-corbos/linux-built-on-ubuntu/free?utm_source=Canonical"><b>Try EB Corbos Linux for free now&nbsp;&rsaquo;</b></a>
         </p>
       </div>
-      <div class="col-6 u-hide--medium u-hide--small u-align--center">
-        {{ image(url="https://assets.ubuntu.com/v1/ce518a18-CoF-2022_solid+O.svg",
-                alt="",
-                width="208",
-                height="200",
-                hi_def=True,
-                loading="lazy") | safe
-        }}
+      <div class="col-6">
+        <div class="u-align--center">
+          {{ image(url="https://assets.ubuntu.com/v1/24713d12-tuv_sud_logo.webp",
+          alt="",
+          width="208",
+          height="200",
+          hi_def=True,
+          loading="lazy") | safe
```

Comment:
```suggestion
          loading="auto") | safe
```
I see this image is immediately visible in the viewport on some devices (tablets etc), so it's better to keep it as auto